src/main/java/com/cinema/util/PasswordValidator.java

```diff
@@ -0,0 +1,26 @@
+package com.cinema.util;
+
+import com.cinema.lib.PasswordValidation;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import org.springframework.beans.BeanWrapperImpl;
+
+public class PasswordValidator implements ConstraintValidator<PasswordValidation, Object> {
+    private String field;
+    private String fieldMatch;
+
+    public void initialize(PasswordValidation constraintAnnotation) {
+        this.field = constraintAnnotation.field();
+        this.fieldMatch = constraintAnnotation.fieldMatch();
+    }
+
+    public boolean isValid(Object value,
```

Comment:
you can inline it if checkstyle allows
```suggestion
    public boolean isValid(Object value, ConstraintValidatorContext context) {
```

Reply:
done